internal/api/handlers.go

```diff
@@ -25,6 +29,10 @@ func NewGoCertRouter(env *Environment) http.Handler {
 	v1 := http.NewServeMux()
 	v1.HandleFunc("GET /status", HealthCheck)
 	v1.Handle("/api/v1/", http.StripPrefix("/api/v1", router))
+	reg := prometheus.NewRegistry()
+	reg.MustRegister(collectors.NewGoCollector())
+	prometheusHandler := promhttp.HandlerFor(reg, promhttp.HandlerOpts{})
```

Comment:
Code relevant to prometheus should be abstracted to its own internal function. It would be nicer to have a function simply return `http.Handler`, which promhttp.HandlerFor already returns, so we can keep the metrics uncoupled.